templates/web/base/report/display.html

```diff
@@ -57,6 +57,8 @@
               c.uri_for( '/report/new', { longitude => longitude, latitude => latitude } )
           ) %]
     </p>
+[% ELSIF problem.cobrand_data == 'noise' %]
+    <p><a href="/noise/existing">Log a new occurrence against your existing report</a></p>
```

Comment:
They're private reports, so will have to be logged in, and they'll have to either be problem reporter or staff user. I guess for a staff user it could say "Log a new occurrence against this report", if that's what you mean?

Reply:
Ah right, that makes sense. I'd not clocked the private bit :|